api/comments/serializers.py

```diff
@@ -26,17 +28,19 @@ class CommentSerializer(JSONAPISerializer):
         'deleted',
         'date_created',
         'date_modified',
+        'page',
         'target'
     ])
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
     content = AuthorizedCharField(source='get_content', required=True, max_length=osf_settings.COMMENT_MAXLENGTH)
+    page = ser.CharField(read_only=True)
```

Comment:
OK, I see. It might be useful to document this on the model field. Also, It would be good to add validation here (you could use a `ChoiceField` here); this will have the ancillary benefit of self-documentation.